configs/components/jsbach/jsbach.yaml

```diff
@@ -118,7 +102,7 @@ choose_version:
                         false:
                                 namelist_dir: "${general.esm_namelist_dir}/jsbach/${version}/${echam.scenario}/no_dynveg"
                                 jsbachnc_file_ending: "${no_dynveg_file_ending}"
-                                dynveg: "_no_dynveg"
+                                dynveg: "_no-dynveg"
```

Comment:
This discrepancy really needs to go away at some point; please check as it might be badly named between Levante and Ollie.

Reply:
the discrepancy is gone :)